src/libstd/io/mod.rs

```diff
@@ -322,6 +587,21 @@ pub trait Write {
     ///
     /// It is **not** considered an error if the entire buffer could not be
     /// written to this writer.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io::Write;
```

Comment:
To stay in line with the examples on `Read`, could this import `std::io::prelude::*`? The `by_ref` probably wants to stay as_is, but everything else should be good to use the prelude.